images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -337,3 +338,22 @@ function Get-PipxVersion {
     $pipxVersion = pipx --version
     return "Pipx $pipxVersion"
 }
+
+function Build-PackageManagementEnvironmentTable {
+    return @(
+        @{
+            "Name" = "CONDA"
+            "Value" = $env:CONDA
+        },
+        @{
+            "Name" = "VCPKG_INSTALLATION_ROOT"
+            "Value" = $env:VCPKG_INSTALLATION_ROOT
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-Path-With-Link($_.Value)
```

Comment:
I guess we don't need to use the function here. These folders are not symlinked.
It makes sense to put symlinks only for Android variables

Reply:
well, i thought we were not able to predict either the path to be symlinked or not so it would be safe to wrap every path with the function instead of patching the specific variable everytime

Reply:
By default, we don't need to print symlinks target. Even path is symlink, we don't print it. The single case where we really need symlink targets is Android `ndk-bundle` folder because it is important for customers where this link points to